tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -59,9 +60,14 @@ sub run {
 
     # Test Maintenance Mode
     foreach my $mod (1, 0) {
-        exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
+        my $retval = exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
```

Comment:
Is this actually catching the errors? AFAIR command is always returning 0, even on failures. I will check the relevant verification runs, but I think more is needed.